lib/frontend/app.js

```diff
@@ -63,6 +63,12 @@ module.exports = (appTokenGenerator) => {
   app.set('view engine', 'hbs');
   app.set('views', path.join(rootPath, 'views'));
 
+  // Handlebars helpers
+  const hbs = require('hbs');
```

Comment:
typically we `require` modules at the top of the file, is there a pre-loading reason to avoid requiring this until within this exported function?